### ISSUE TITLE
Error opening complete form in IE11

### ISSUE THREAD

**Issue by pbowen-oc** [OpenClinica/enketo-express-oc#375]
We are getting an error loading any complete form in IE11:
![Annotation 2020-09-05 020543](https://user-images.githubusercontent.com/25179620/92299123-a1734e80-ef1d-11ea-9070-d4f7a4b12ae7.png)

This is image is from view:
/edit/fs/rfc/i/

XForm:
[IE11 complete xform.txt](https://github.com/OpenClinica/enketo-express-oc/files/5177610/IE11.complete.xform.txt)

Instance:
[IE11 complete instance.txt](https://github.com/OpenClinica/enketo-express-oc/files/5177611/IE11.complete.instance.txt)

**Comment by MartijnR** [comment]
```bash
curl --user enketorules: -d "server_url=http://localhost:3000&form_id=oc-340&ecid=1&instance=\
<data xmlns:oc=\"http://openclinica.org/xforms\" oc:complete=\"true\"   xmlns:enk=\"http://enketo.org/xforms\" >\
    <a>aa</a>
    <b>aa</b>
    <meta>
        <instanceID>123</instanceID>
    </meta>
</data>\
&instance_id=123" http://localhost:8005/oc/api/v1/instance/edit/rfc
```

**Comment by MartijnR** [comment]
Thankfully this could be fixed outside of enketo-core. It is now fixed in `2.3.8.oc-5`.

**Comment by pbowen-oc** [comment]
Looks good after testing.
